app/models/my_module.rb

```diff
@@ -313,7 +313,7 @@ def repository_json_hot(repository, order)
     rows = repository.assigned_rows(self).includes(:created_by).order(created_at: order)
     rows.find_each do |row|
       row_json = []
-      row_json << (row.repository.is_a?(RepositorySnapshot) ? row.parent_id : row.id)
+      row_json << (row.parent&.code || row.code)
```

Comment:
So it will show code of the new item in the snapshot for deleted inventory instead of original ID?